smartseq2_single_sample/ss2_single_sample.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I still think calling this file something like `ss2_single_sample_example.json` would be helpful.